### ISSUE TITLE
Gem crashes with syntax errors

### ISSUE THREAD

**Issue by MatthewRalston** [afaundez/jekyll-favicon#1]
Yikes, totally crashed.
```bash
~/Projects/blog>jekyll serve
Configuration file: /Users/Matthew/Projects/blog/_config.yml
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll-favicon.rb:5:in `require_relative': /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:11: syntax error, unexpected << (SyntaxError)
        <<~HTML
          ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:12: syntax error, unexpected '<', expecting keyword_end
        <!-- Begin Jekyll Favicon tag v#{Jekyll::Favicon::VERSION} -->
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:12: syntax error, unexpected tCONSTANT, expecting keyword_do or '{' or '('
        <!-- Begin Jekyll Favicon tag v#{Jekyll::Favicon::VERSION} -->
                         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:13: syntax error, unexpected '<', expecting keyword_end
        <link href="/apple-touch-icon....
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:13: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...ref="/apple-touch-icon.png" rel="apple-touch-icon" sizes="18...
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:13: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...g" rel="apple-touch-icon" sizes="180x180">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:14: syntax error, unexpected '<'
        <link href="/favicon-32x32.png...
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:14: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...k href="/favicon-32x32.png" rel="icon" sizes="32x32" type="i...
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:14: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...con-32x32.png" rel="icon" sizes="32x32" type="image/png">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:14: syntax error, unexpected tIDENTIFIER, expecting keyword_end
..." rel="icon" sizes="32x32" type="image/png">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:15: syntax error, unexpected '<'
        <link href="/favicon-16x16.png...
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:15: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...k href="/favicon-16x16.png" rel="icon" sizes="16x16" type="i...
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:15: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...con-16x16.png" rel="icon" sizes="16x16" type="image/png">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:15: syntax error, unexpected tIDENTIFIER, expecting keyword_end
..." rel="icon" sizes="16x16" type="image/png">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:16: syntax error, unexpected '<'
        <link href="/manifest.json" rel="manifest">
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:16: syntax error, unexpected tIDENTIFIER, expecting keyword_end
        <link href="/manifest.json" rel="manifest">
                                       ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:17: syntax error, unexpected '<'
        <link color="#5bbad5" href="/s...
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:17: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...     <link color="#5bbad5" href="/safari-pinned-tab.svg" rel...
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:17: syntax error, unexpected tIDENTIFIER, expecting keyword_end
...ef="/safari-pinned-tab.svg" rel="mask-icon">
...                               ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:18: syntax error, unexpected '<'
        <meta content="#ffffff" name="theme-color" />
         ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:18: syntax error, unexpected tIDENTIFIER, expecting keyword_end
        <meta content="#ffffff" name="theme-color" />
                                    ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:18: syntax error, unexpected '>'
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:19: syntax error, unexpected tCONSTANT, expecting keyword_do or '{' or '('
        <!-- End Jekyll Favicon tag -->
                       ^
/Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll/tags/favicon.rb:19: syntax error, unexpected '\n', expecting keyword_do_LAMBDA or tLAMBEG
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-favicon-0.1.0/lib/jekyll-favicon.rb:5:in `<top (required)>'
	from /Users/Matthew/.rvm/rubies/ruby-2.1.2/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:128:in `require'
	from /Users/Matthew/.rvm/rubies/ruby-2.1.2/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
	from /Users/Matthew/.rvm/rubies/ruby-2.1.2/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/external.rb:58:in `block in require_with_graceful_fail'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/external.rb:55:in `each'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/external.rb:55:in `require_with_graceful_fail'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/plugin_manager.rb:30:in `require_gems'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/plugin_manager.rb:22:in `conscientious_require'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/site.rb:113:in `setup'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/site.rb:34:in `initialize'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/build.rb:30:in `new'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/build.rb:30:in `process'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:93:in `block in start'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:93:in `each'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:93:in `start'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `call'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/gems/jekyll-3.7.3/exe/jekyll:15:in `<top (required)>'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/bin/jekyll:23:in `load'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/bin/jekyll:23:in `<main>'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
	from /Users/Matthew/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by afaundez** [comment]
Hi @MatthewRalston,

Thanks for trying out this gem. I was able to reproduce your situation: the problem was the `<<~` heredoc it doesn't exist on ruby 2.1.2. I already fix it, please update to 0.1.1

```ruby
gem 'jekyll-favicon', '~> 0.1.1', group: :jekyll_plugins
```

I also found an error in the usage instructions (47d4bebb5a7c6fcaafae6f37ae12aafe235e7d4c)

**Comment by MatthewRalston** [comment]
Thank you sir, I'll give this a try.
